docs/sections_readme/case_studies.md

```diff
@@ -0,0 +1,58 @@
+## Case studies
```

Comment:
It could be that sphinx will complain about the headlines. Check the logs, but if you split up the README, you could remove one hashtag per headline...

That said, the website looks good, so no big issue